services/harmony/app/frontends/workflow-ui.ts

```diff
@@ -107,8 +107,8 @@ function parseQuery( /* eslint-disable @typescript-eslint/no-explicit-any */
       .filter(option => isAdminAccess && /^user: [A-Za-z0-9\.\_]{4,30}$/.test(option.value));
     const userValues = validUserSelections.map(option => option.value.split('user: ')[1]);
     const validProviderSelections = selectedOptions
-      .filter(option => isAdminAccess && /^prov: [A-Za-z0-9_]{1,100}$/.test(option.value));
-    const providerValues = validProviderSelections.map(option => option.value.split('prov: ')[1].toLowerCase());
+      .filter(option => isAdminAccess && /^provider: [A-Za-z0-9_]{1,100}$/.test(option.value));
```

Comment:
I didn't realize the provider filter was only on /admin/workflow-ui page. I think it could be useful for some people on the /workflow-ui page as well. Is there any reason not to include it there?